src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h

```diff
@@ -171,5 +172,9 @@ namespace OpenMS
     std::vector<double> stn_estimates_;
   };
 
+  /// Picks @p n_scans from level 1 randomly and returns either average intensity at a certain @p percentile.
+  /// If no MS1 scans are present, 0.0 is returned
+  OPENMS_DLLAPI float estimateNoiseFromRandomMS1Scans(const MSExperiment& exp, const UInt n_scans = 10, const double percentile = 80);
```

Comment:
pass leve to make it more genericl?

Reply:
possible, but this is just a refactor.
For MS2 its also less useful since MS2 is much less affected by noise.
But could be done.

Reply:
done